AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -605,8 +605,45 @@ - (void)collectionView:(UICollectionView *)collectionView didEndDisplayingCell:(
   cellNode.scrollView = nil;
 }
 
-#pragma mark -
-#pragma mark Scroll Direction.
+
+- (void)scrollViewDidScroll:(UIScrollView *)scrollView
+{
+  // If a scroll happenes the current range mode needs to go to full
```

Comment:
Yes I changed the grouping of the methods to have the same grouping in ASCollectionView and ASTableView. Before this was not the case. Furthermore this is also in preparation for further consolidating of logic of ASCollectionView and ASTableView.